internal/blsgen/lib.go

```diff
@@ -28,16 +35,10 @@ func toISO8601(t time.Time) string {
 		t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond(), tz)
 }
 
-func keyFileName(publicKey *ffi_bls.PublicKey) string {
-	ts := time.Now().UTC()
-	serializedPublicKey := publicKey.SerializeToHexStr()
-	return fmt.Sprintf("UTC--%s--bls_%s", toISO8601(ts), serializedPublicKey)
-}
-
 // WritePrivateKeyWithPassPhrase encrypt the key with passphrase and write into disk.
 func WritePrivateKeyWithPassPhrase(privateKey *ffi_bls.SecretKey, passphrase string) string {
 	publickKey := privateKey.GetPublicKey()
-	fileName := keyFileName(publickKey)
+	fileName := publickKey.SerializeToHexStr() + ".key"
```

Comment:
the filename is hardcoded.,  This info should be available to user so that people know this constraint.